packages/components/src/Actions/ActionFile/ActionFile.component.js

```diff
@@ -133,7 +135,7 @@ class ActionFile extends React.Component {
 					name={name}
 					id={localId}
 					disabled={inProgress || disabled}
-					className={classNames(theme['action-file-input'], 'sr-only')}
+					className={classNames(theme['action-file-input'], 'sr-only', className)}
```

Comment:
should stay `classNames` since you can pass many class name to this props, you can however alias the `classNames` function on import.